streams/src/main/java/org/apache/kafka/streams/state/internals/metrics/NamedCacheMetrics.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.streams.state.internals.metrics;
+
+import org.apache.kafka.common.metrics.Sensor;
+import org.apache.kafka.streams.processor.internals.metrics.StreamsMetricsImpl;
+
+import static org.apache.kafka.streams.processor.internals.metrics.StreamsMetricsImpl.CACHE_LEVEL_GROUP;
+import static org.apache.kafka.streams.processor.internals.metrics.StreamsMetricsImpl.ROLLUP_VALUE;
+import static org.apache.kafka.streams.processor.internals.metrics.StreamsMetricsImpl.Version.FROM_100_TO_23;
+import static org.apache.kafka.streams.processor.internals.metrics.StreamsMetricsImpl.addAvgAndMinAndMaxToSensor;
+
+public class NamedCacheMetrics {
+    private NamedCacheMetrics() {}
+
+    private static final String HIT_RATIO_0100_TO_23 = "hitRatio";
+    private static final String HIT_RATIO = "hit-ratio";
+    private static final String HIT_RATIO_AVG_DESCRIPTION = "The average cache hit ratio";
```

Comment:
nit: AVG_VALUE_DOC + "cacht hit ratio" ?

Reply:
I would like to not let different descriptions depend too much from each other, since that tends to make changes to descriptions tedious and does not improve too much the footprint of the descriptions. Additionally, I plan to consolidate methods like `addAvgAndMaxToSensor()` to take descriptions as parameters in the next PRs. Thus, I would like to leave that as it is.

Reply:
SG.